components/TextAreaDetails.tsx

```diff
@@ -29,13 +37,53 @@ export const TextAreaDetails: React.FunctionComponent<Props> = props => {
     label,
     contentBeforeTextArea,
     contentAfterTextArea,
+    includeCheckbox,
     value,
     onValueChange,
     required,
     disabled
   } = props;
 
   const summaryId = `${name}-summary`;
+  let postVisitActionCheckbox = null;
+  if (includeCheckbox) {
+    const checkbox = {
+      label: "Create a post-visit action",
+      value: true
+    };
+    const id = `${summaryId}-post-visit-action`;
+
+    const checkboxItem: CheckboxItem[] = [
+      {
+        id,
+        value: `${checkbox.value}`,
+        label: { id: `${id}-label`, children: checkbox.label },
+        checked: value.isPostVisitAction === checkbox.value,
+        disabled
+      }
+    ];
+
+    postVisitActionCheckbox = (
+      <LBHCheckboxes
+        name={name}
+        items={checkboxItem}
+        onChange={(checkboxValue: string[]): void => {
+          if (checkboxValue[0] === "true") {
+            return onValueChange({
+              value: value.value,
+              isPostVisitAction: true
+            });
+          } else {
+            return onValueChange({
+              value: value.value,
+              isPostVisitAction: false
+            });
```

Comment:
This could be simplified to:

```
            return onValueChange({
              value: value.value,
              isPostVisitAction: checkboxValue[0] === "true"
            });
```

Reply:
Or even:

```
            return onValueChange({
              value: value.value,
              isPostVisitAction: checkboxValue.length > 0
            });
```